pkg/server/commands/write_authzmodel_test.go

```diff
@@ -0,0 +1,70 @@
+package commands
+
+import (
+	"context"
+	"testing"
+
+	"github.com/oklog/ulid/v2"
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/mock/gomock"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	mockstorage "github.com/openfga/openfga/internal/mocks"
+	"github.com/openfga/openfga/pkg/typesystem"
+)
+
+func TestWriteAuthorizationModelWithExperimentalEnableModularModels(t *testing.T) {
+	ctx := context.Background()
```

Comment:
```suggestion
	t.Cleanup(func() {
		goleak.VerifyNone(t)
	})
	ctx := context.Background()
```

Reply:
Added in 3d14bf43dc9e2d0b6ad2a4f93503c0d4cf2f0de6